### PR TITLE
Documentation refactor

### PR DESCRIPTION
**Pull request by RomanKosovnenko**
SAML configuration documentation in a separate file

### CODE REVIEW

File: SAMLconfiguration.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
+# SAML
+
+Greenlight is a Service Provider, that connects to IdP to get authentification.
+
+Unfortunatly, Greenlight not supported SAML out of the box. But there is a [PR Request](https://github.com/bigbluebutton/greenlight/pull/1334) thet gives needed functionality.
```
Comment (K0nstantine) [inline SAMLconfiguration.md:5]: does not support SAML

File: SAMLconfiguration.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+<md:SingleLogoutService Binding="urn:oasis:names:tc:SAML:2.0:bindings:HTTP-Redirect" Location="http://localhost:8080/simplesaml/saml2/idp/SingleLogoutService.php"/>
+<md:NameIDFormat>urn:oasis:names:tc:SAML:2.0:nameid-format:transient</md:NameIDFormat>
+<md:SingleSignOnService Binding="urn:oasis:names:tc:SAML:2.0:bindings:HTTP-Redirect" Location="http://localhost:8080/simplesaml/saml2/idp/SSOService.php"/>
+</md:IDPSSODescriptor>
+</md:EntityDescriptor>
+```
+## SAML configuration
+SAML configuration required configuration from both sides. 
+1. Greenlight required uniq identifier of SP, that should be stored in IdP. 
+The best way is to set SAML_ISSUER variable in .env and then lookup greenlight metadata from IpD.
```
Comment (K0nstantine) [inline SAMLconfiguration.md:58]: ... from I**dP**

File: SAMLconfiguration.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+
+2. Set SAML_IDP_URL variable. SAML_IDP_URL is the URL to which the authentication request should be sent. This would be on the identity provider. It can be found in the IDP's metadata in the <md:SingleSignOnService> tag. Get this tag from IdP metadata.
+
+3. SAML_IDP_CERT_FINGERPRINT is the fingerprint of the certificate used by the IDP in sha1, for example "25:72:85:66:C9:94:22:98:36:84:11:E1:88:C7:AC:40:98:F9:E7:82"(without ").
```
Comment (K0nstantine) [inline SAMLconfiguration.md:62]: ... I**d**P ...     be consistent ;)

File: SAMLconfiguration.md [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+## SAML configuration
+SAML configuration required configuration from both sides. 
+1. Greenlight required uniq identifier of SP, that should be stored in IdP. 
+The best way is to set SAML_ISSUER variable in .env and then lookup greenlight metadata from IpD.
+
+2. Set SAML_IDP_URL variable. SAML_IDP_URL is the URL to which the authentication request should be sent. This would be on the identity provider. It can be found in the IDP's metadata in the <md:SingleSignOnService> tag. Get this tag from IdP metadata.
+
+3. SAML_IDP_CERT_FINGERPRINT is the fingerprint of the certificate used by the IDP in sha1, for example "25:72:85:66:C9:94:22:98:36:84:11:E1:88:C7:AC:40:98:F9:E7:82"(without "). 
+4. SAML_NAME_IDENTIFIER - could get from IdP metadata. by default it is "urn:oasis:names:tc:SAML:2.0:nameid-format:transient"
+5. All others variables needed to map SAML response fields to user fields in the Greenlight.
```
Comment (K0nstantine) [inline SAMLconfiguration.md:64]: All other variables ... to **the** user fields...

File: SAMLconfiguration.md [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+To get all available fields check IdP or record network activity in developer console in your browser. Try to sign in using SAML, copy encoded SAML Response, decode it using [decoder](https://www.samltool.com/decode.php) and map values from responce by name fields to variables in .env
```
Comment (K0nstantine) [inline SAMLconfiguration.md:65]: ... map values from **the** respon**s**e to the variables in .env by the fields' name...

File: SAMLconfiguration.md (hunk 1 above)
Comment (RomanKosovnenko) [reply SAMLconfiguration.md]: e8ec4a41be7bcc793f43b41ac3c11719c81ab095

File: SAMLconfiguration.md (hunk 2 above)
Comment (RomanKosovnenko) [reply SAMLconfiguration.md]: e8ec4a41be7bcc793f43b41ac3c11719c81ab095

File: SAMLconfiguration.md (hunk 3 above)
Comment (RomanKosovnenko) [reply SAMLconfiguration.md]: e8ec4a41be7bcc793f43b41ac3c11719c81ab095

File: SAMLconfiguration.md (hunk 4 above)
Comment (RomanKosovnenko) [reply SAMLconfiguration.md]: e8ec4a41be7bcc793f43b41ac3c11719c81ab095

File: SAMLconfiguration.md (hunk 5 above)
Comment (RomanKosovnenko) [reply SAMLconfiguration.md]: e8ec4a41be7bcc793f43b41ac3c11719c81ab095

**Pull request merged**